tests/integration/test_integration.py

```diff
@@ -2674,3 +2677,56 @@ def test_binding_constraint_manager(app: FastAPI):
     constraints = binding_constraint["constraints"]
     assert res.status_code == 200
     assert constraints is None
+
+
+def test_import(app: FastAPI) -> None:
+    client, admin_credentials = init_test(app)
```

Comment:
Note: `client` and `admin_credentials` are also available as pytest fixtures, so that we can pass them as parameters and remove that line. It should work as-is 😋